### PR TITLE
Allow resources to have icons in Concourse UI

### PR DESCRIPTION
**Pull request by idrop**
Make our resources have icons like this:

Screen Shot 2019-04-26 at 15 42 56

### CODE REVIEW

File: templates/build.yaml [hunk 1]
```diff
@@ -64,91 +64,107 @@ spec:
 
     resources:
     - name: gateway-repo
+      icon: gate
       type: github
       source:
         <<: *github_source
 
     - name: translator-repo
+      icon: translate
```
Comment (CharlesIC) [inline templates/build.yaml]: Maybe use `file-xml` for the Translator?

File: templates/build.yaml [hunk 2]
```diff
@@ -64,91 +64,107 @@ spec:
...
         <<: *github_source
 
     - name: parser-repo
+      icon: page-next
       type: github
       source:
         <<: *github_source
 
     - name: stub-connector-repo
+      icon: pipe-disconnected
```
Comment (CharlesIC) [inline templates/build.yaml]: Maybe use `power-plug`?

File: templates/build.yaml [hunk 3]
```diff
@@ -64,91 +64,107 @@ spec:
...
       type: github
       source:
         <<: *github_source
 
     - name: cloudhsm-repo
+      icon: book-lock
```
Comment (CharlesIC) [inline templates/build.yaml]: What about `folder-key-network` or `shield-key`?

File: templates/build.yaml [hunk 4]
```diff
@@ -64,91 +64,107 @@ spec:
...
       type: github
       source:
         <<: *github_source
 
     - name: verify-service-provider-repo
+      icon: account-check
```
Comment (CharlesIC) [inline templates/build.yaml]: `marker-check` looks almost like the Verify logo 😅

File: templates/build.yaml (hunk 1 above)
Comment (idrop) [reply templates/build.yaml]: Updated to your suggestion

File: templates/build.yaml (hunk 4 above)
Comment (idrop) [reply templates/build.yaml]: Updated to your suggestion

File: templates/build.yaml (hunk 3 above)
Comment (idrop) [reply templates/build.yaml]: Updated to your suggestion

File: templates/build.yaml (hunk 2 above)
Comment (idrop) [reply templates/build.yaml]: Updated to your suggestion

**Pull request merged**